Standardization

karikarshivani opened the pull request:
### [Report](https://github.com/karikarshivani/InSpecReports/blob/master/Standardization/aws-rds-mysql_04272021A.json) before changes
---
**Standardization Checklist**
- [x] README - [model](https://github.com/mitre/docs-mitre-inspec/blob/master/templates/baseline_readme.md)
- [x] in inspec.yml, validate the title and the summary to be consistent
- [x] in inspec.yml, set `inspec_version: ">= 4.0"`
- [x] in inspec.yml, validate if any supports statements are valid
- [x] in inspec.yml, change from attributes to inputs
- [x] in inspec.yml, set `sensitive: ` flags as appropriate, e.g., for user names, passwords, hostnames, etc.
- [x] Update the License.md
---
### [Report](https://github.com/karikarshivani/InSpecReports/blob/master/Standardization/aws-rds-mysql_04272021B.json) after changes

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

karikarshivani committed:
Updated License.md

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

karikarshivani committed:
Updated inspec.yml to the latest standards

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

ejaronne commented:
When re-iterating the sensitive inputs in the inspec.yml, weirdly enough *do not* use the profile: {name of baseline}, otherwise it won't "stick" in the json output.

For example, instead of:

  - name: password
    description: 'password MySQL DB Server'
    type: string
    value: ''
    **_profile: oracle-mysql-ee-5.7-cis-baseline_**
    sensitive: true

use:

  - name: password
    description: 'password MySQL DB Server'
    type: string
    value: ''
    sensitive: true

This may have affected other profiles already moved to done.

karikarshivani committed:
Removed profile overrides

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

karikarshivani commented:
> When re-iterating the sensitive inputs in the inspec.yml, weirdly enough _do not_ use the profile: {name of baseline}, otherwise it won't "stick" in the json output.
> 
> For example, instead of:
> 
> * name: password
>   description: 'password MySQL DB Server'
>   type: string
>   value: ''
>   **_profile: oracle-mysql-ee-5.7-cis-baseline_**
>   sensitive: true
> 
> use:
> 
> * name: password
>   description: 'password MySQL DB Server'
>   type: string
>   value: ''
>   sensitive: true
> 
> This may have affected other profiles already moved to done.

Alright, I have removed the profile keys that I added. The first two inputs already had it, okay to leave them as they are?

karikarshivani commented:
![ReportComparison](https://user-images.githubusercontent.com/11844975/116601394-77c9c600-a8f8-11eb-8c33-93b1d81a1b47.png)

HackerShark committed:
Updating README.md to latest standards

Signed-off-by: HackerShark <melsharkawi@mitre.org>

HackerShark committed:
Fixing issues in README.md

Signed-off-by: HackerShark <melsharkawi@mitre.org>
